app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -152,33 +153,34 @@ class SupplementalApplicationPage extends React.Component {
   }
 
   render () {
-    const { statusHistory, fileBaseUrl, application, availableUnits } = this.props
+    const { statusHistory, fileBaseUrl, availableUnits } = this.props
     const {
       confirmedPreferencesFailed,
       amis,
       amiCharts,
       statusModal,
-      loading
+      loading,
+      persistedApplication
```

Comment:
@akegan Is this perhaps the same move of application from props to state that you and I discussed at end of yesterday on your lease preference used PR?

Reply:
looks like it might be! I'll sync with fed on this now